README.md

```diff
@@ -89,3 +89,18 @@ left column and its corresponding action in the right column. The formatting is
 clean and easy to read.
 
 Our CI enforces formatting, linting, and tests.
+
+## Changelog
+
+Check the [GitHub Releases](https://github.com/higlass/higlass-python/releases) for a detailed changelog.
+
+## Release
+
+Releases are managed via the GitHub UI:
+
+1. [Draft a new release](https://github.com/higlass/higlass-python/releases/new)
+2. **Create a tag** in the format `v[major].[minor].[patch]`.
```

Comment:
As per the Slack conversation, I recommend making 2. a sub item of 1 so that it's clear that the tag is created during the release process.

Reply:
Since all the steps that follow are a part of "drafting a release", I reorganized things. Let me know hwat you think.

Reply:
Looks good 👍